pkg/loadbalancer/loadbalancer.go

```diff
@@ -742,32 +742,61 @@ func (l L4Addr) String() string {
 	return fmt.Sprintf("%d/%s", l.Port, l.Protocol)
 }
 
-// L3n4Addr is used to store, as an unique L3+L4 address in the KVStore. It also
-// includes the lookup scope for frontend addresses which is used in service
-// handling for externalTrafficPolicy=Local and internalTrafficPolicy=Local,
-// that is, Scope{External,Internal}.
-//
-// +deepequal-gen=true
-// +deepequal-gen:private-method=true
-// +k8s:deepcopy-gen=true
-type L3n4Addr struct {
-	AddrCluster cmtypes.AddrCluster
+// L3n4Addr is an unique L3+L4 address and scope (for traffic policies).
+type L3n4Addr unique.Handle[l3n4AddrRep]
```

Comment:
It's more a question since I don't know that much this part of the code but AFAIU there is no cleanup of old/no longer used `l3n4AddrRep` so what prevent this for growing somewhat indefinitely as long as the agent and anywhere else this is used is not restarted?